ST_mortality.c

```diff
@@ -588,29 +652,39 @@ static void _succulents( const SppIndex sp) {
 }
 
 
-/***********************************************************/
+/**
+ * \brief Implements slow growth mortality.
+ * 
+ * Kill plants based on a probability if the growth rate
+ * is less than the "slow rate" which is defined by the
+ * user in the group-level parameters (max_slow) and in
+ * the species-level parameters (max_rate). The slow rate
+ * is growthrate <= max_slow * max_rate.
+ *
+ * Increment the counter for number of years of slow growth.
+ * If the number of years of slow growth is greater than
+ * max_slow (defined in species.in), draw a random number
+ * and test it against the probability of mortality.  C&L'90
+ * defines this value as a constant, but it might be better
+ * to define it in the groups or species parameters.
+ *
+ * Of course, annuals aren't subject to this mortality,
+ * nor are new plants.
```

Comment:
is this comment regarding "new" plants true? I'm not sure that is correct.

Reply:
There is a line in the function that skips the loop for new plants: `if ( ndv->age == 1) continue;`. However, "new" is a relative term, so I will change the comment to "... nor are 1 year old plants"